src/components/OsnapV2/TvsChip.tsx

```diff
@@ -0,0 +1,18 @@
+import { GradientBorder, GradientBorderProps } from "@/components/GradientBorder";
+
+type TvsChipProps = GradientBorderProps & {
+  amount: number; // eg. 83_000_000
+};
+
+export function TvsChip({ amount, className, ...props }: TvsChipProps) {
+  const formattedAmount = (amount / 10 ** 6).toLocaleString();
```

Comment:
yea i wouldnt even bother with this. lets just pass through the env as a string where we need to know we are putting in a value like `${amount}+` so for example the env would need to be `83M` or `1.2B` or any kind of number `100,230.99` where the user just formats it how they want